recipes/aravis/all/conanfile.py

```diff
@@ -87,10 +103,14 @@ def validate(self):
             )
 
     def build_requirements(self):
-        self.tool_requires("meson/1.2.1")
+        #windows build: meson/1.2.1 works, meson/1.2.2 breaks for some reason!
+        if self.settings.os == "Windows":
+            self.tool_requires("meson/[<1.2.2]")
+        else:
+            self.tool_requires("meson/1.2.2")
```

Comment:
```suggestion
        self.tool_requires("meson/1.2.1")
```

Let's keep it pinned for now - We expect to enable tool require version ranges soon, but this is a better fit for now